streams/src/test/java/org/apache/kafka/streams/processor/internals/DefaultStateUpdaterTest.java

```diff
@@ -652,7 +876,10 @@ private void verifyUpdatingTasks(final Task... tasks) throws Exception {
                 "Did not get all updating task within the given timeout!"
             );
             assertEquals(expectedUpdatingTasks.size(), updatingTasks.size());
-            assertTrue(updatingTasks.stream().allMatch(task -> task.state() == State.RESTORING));
+            assertTrue(updatingTasks.stream()
+                .allMatch(task -> task.isActive() && task.state() == State.RESTORING
+                    ||
+                    !task.isActive() && task.state() == State.RUNNING));
```

Comment:
Here I missed to verify the updating standby tasks before.

Reply:
Good call.